src/app/containers/Columns/ShowBookmarkAddColumn.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import { columnNameOnchange, addColumn } from '../../actions/'
+import BookmarkAddColumn from '../../components/Columns/BookmarkAddColumn'
+
+const mapStateToProps = state => {
+  return { value: state.bookmarks.name,
+    columns: state.bookmarks.columns }
+}
+
+@connect(mapStateToProps)
```

Comment:
the second paramter you can pass an object of actions aka `mapDispatchToProps`

```javascript
// ...
const mapDispatchToProps = {
  addColumn
}

@connect(mapStateToProps, mapDispatchToProps)
class Component from React.Component { /* ... */ } 

/* somewhere in the component */
this.props.addColumn()
```

so you call the function as a property of a component, but he will execute the function over `store.dispatch`